### DISCUSSION TITLE
역의 활성화 상태 구분

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#15] [Ideas]
- 색깔
- 명도 (밝기)
- 투명도
- 기타

의견 바랍니다

**Comment by seong0929** [comment]
![image](https://user-images.githubusercontent.com/50050003/202691620-38a146d9-860b-4ab1-9795-779111a15d3a.png)

| | 내부 | 테두리 |
|---|---|---|
|활성화 역 | RGBA(255,255,255,1) | RGBA(0,0,0,1) |
|비활성화 역 | RGBA(255,255,255,1)  | RGBA(192,192,192,1) |
|파괴된 역 |  RGBA(255,128,128,0.5) | RGBA(0,0,0,1) |

**Reply by seong0929** [reply]
![image](https://user-images.githubusercontent.com/50050003/205592052-befc5269-94fd-4234-8241-786a6fb0f560.png)
![image](https://user-images.githubusercontent.com/50050003/205592236-684c1b20-7861-4fc5-8bfe-24914d94ddb6.png)

| | 내부 | 테두리 |
|---|---|---|
|활성화 역 | RGB(1,1,1), HSV(0,0,1) | RGB(0,0,0),HSV(0,0,0) |
|비활성화 역 |RGB(1,1,1), HSV(0,0,1)  | RGB(0.21875,0.21875,0.21875),HSV(0,0,0.21875) |
|파괴된 역 |  RGB(0.5,0.25,0.25),HSV(0,0.5,0.5) | RGB(0,0,0),HSV(0,0,0) |
